packages/jest-types/src/Config.ts

```diff
@@ -289,6 +289,7 @@ export type InitialOptions = Partial<{
   runTestsByPath: boolean;
   runtime: string;
   sandboxInjectedGlobals: Array<string>;
+  seed: number;
```

Comment:
Documentation should be added to [Configuration.md](https://github.com/facebook/jest/blob/main/docs/Configuration.md) or I missed something?

Reply:
That wasn't meant to be there. I'll delete it from the initial options